steve/util.py

```diff
@@ -216,18 +391,21 @@ def get_project_config_file_name():
     return SteveConfig.file_name
 
 
-def create_project_config_file(dir_path):
+def create_project_config_file(dir_path, cfg=None):
```

Comment:
As a rule of thumb, you should have immutable arg defaults because it's better for self-documentation. In this case, it's immutable, so that's fine. You're right in that it's bad to have mutable defaults.